.github/workflows/pack-and-release.yml

```
@@ -0,0 +1,293 @@
+
+      - name: Create GitHub release
+        uses: ncipollo/release-action@v1
+        with:
+          tag: v${{ steps.gitversion.outputs.SemVer }}
+          name: DMF v${{ steps.gitversion.outputs.SemVer }}
+          commit: ${{ steps.source.outputs.head_sha }}
+          artifacts: ".output/a/archives/**/*.zip"
+          artifactErrorsFailBuild: true
+          generateReleaseNotes: true
+          allowUpdates: true
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)